### DISCUSSION TITLE
Public plans for LTS releases shared in advance with plugin developers

### DISCUSSION THREAD

**Discussion by pevogam** [avocado-framework/avocado#4520] [Q&A]
Hi all,

Are there any plans or partial knowledge about when the next LTS should be expected and which version is most probably going to be tagged as an LTS?

When looking at [the next LTS](https://avocado-framework.readthedocs.io/en/latest/releases/lts/next.html) it doesn't seem like much has been decided yet and considering the distance between LTS versions 52.0-69.0-82.0 I could interpolate ~99.0 with about 15 releases on average between the LTS-es.

**Comment by pevogam** [comment]
To answer my own question, the planned LTS is 92.0 and the planned date is in the first days of October.

**Reply by beraldoleal** [reply]
Sorry, we missed this question.

**Comment by pevogam** [comment]
Hi @clebergnu @beraldoleal @richtja @willianrampazzo is there a way to make the avocado LTS version releases public and known to plugin developers (even if tentative) at least a month or two ahead of time? As developers of a complete external avocado plugin our release process it tightly related to the LTS releases of avocado and we run into the risk of our plugin becoming unusable by anyone if certain pull requests are not merged prior to the next release, yet such release might happen very suddenly and out of our control to help merging the respective pull requests if it is not communicated and simply pushed through.

I would really appreciate any information regarding this because the entire ecosystem of plugins would need to usually sync (and be compatible) at least on major LTS releases and the corresponding maintainers of these plugins would likely appreciate advance communication even if they don't mind giving out control over the release version and time (similarly to Avocado VT). Thanks!
